R/setup.R

```diff
@@ -70,7 +70,8 @@ pbSetup <- function(apikey, conffile, defdev) {
     json <- toJSON(reslist, auto_unbox=TRUE, pretty=TRUE)
 
     f <- file(conffile,open = "w")
-    cat(json,file = f)
+    cat(json, file = f)
+    cat('\n', file = f)
```

Comment:
It was late and I can never remember what the default `sep` is for `cat`. Will combine. Do you prefer squashing commits, or can I just commit on top of this one?